base/steer/FairRootManager.cxx

```diff
@@ -455,11 +456,12 @@ Int_t FairRootManager::ReadEvent(Int_t i)
     fCurrentEntryNo = i;
 
     Int_t readEventResult = fSource->ReadEvent(i);
+    Double_t eventTime = ((FairFileSource*)fSource)->GetEventTime();
 
     fSource->FillEventHeader(fEventHeader);
     fCurrentTime = fEventHeader->GetEventTime();
 
-    LOG(debug) << "--Event number --- " << fCurrentEntryNo << " with time ---- " << fCurrentTime;
+    std::cout << "--Event number --- " << fCurrentEntryNo << " with time ---- " << fCurrentTime << " eventTime " << eventTime << std::endl;
```

Comment:
Why not use the logger here? This will always print the lines regardless of the configured log severity.

Reply:
I will switch it back to the logger. The only reason this was done was because of debugging reasons.

Reply:
ok, thanks. I think eventdisplay/tracks/FairGeoTrackHandler.cxx:19 has the same issue.

Reply:
I changed now all std::cout in folder /eventdisplay to fairlogger output :-)